### PR TITLE
Divyanshu | Test | Convert Setting side bar all tabs into material - Bugs

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cv9wkp0

### CODE REVIEW

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts [hunk 1]
```diff
@@ -416,7 +419,13 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
      */
     public selectLinkedAccount(event: IOption): void {
         if (event?.value) {
-            this.paypalObj.account.name = event.label;
+            this.linkedAccountLabel = event.label;
+            this.paypalAccounts$.subscribe((arr: IOption[]) => {
+                let res = find(arr, (o) => o?.value === event.value);
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.ts]: change single value instance

File: apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts [hunk 2]
```diff
@@ -58,27 +58,98 @@ export class PersonalInformationComponent implements OnInit, OnDestroy {
...
      * @memberof PersonalInformationComponent
      */
     public ngOnInit(): void {
-        this.tempProfileData = this.profileData;
+        this.initProfileForm();
         this.voucherApiVersion = this.generalService.voucherApiVersion;
         this.isValidDomain = this.generalService.checkDashCharacterNumberPattern(this.profileData.portalDomain);
         this.saveProfileSubject.pipe(takeUntil(this.destroyed$)).subscribe((res) => {
-            if (res) {
+            if (res && this.profileForm.dirty) {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts:76]: Check the condition

File: apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts [hunk 3]
```diff
@@ -58,27 +58,98 @@ export class PersonalInformationComponent implements OnInit, OnDestroy {
...
                 this.saveProfile.emit(this.updatedData);
             }
         });
     }
 
+    /**
+    *  Handle On Change of input properties
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts]: correct the documentation this is hook

File: apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts [hunk 4]
```diff
@@ -58,27 +58,98 @@ export class PersonalInformationComponent implements OnInit, OnDestroy {
...
+    * @memberof PersonalInformationComponent
+    */
+    public ngOnChanges(changes: SimpleChanges): void {
+        if (changes?.profileData) {
+            if (this.profileData?.alias || this.profileData?.name) {
+                this.initProfileForm(this.profileData);
+            }
+
+            if (this.organizationType === 'COMPANY') {
+                this.profileForm?.get('name')?.valueChanges?.pipe(takeUntil(this.destroyed$), debounceTime(700)).subscribe((value) => {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts:94]: move to ngOnInit hook

File: apps/web-giddh/src/app/shared/create-address/create-address.component.html [hunk 5]
```diff
@@ -53,25 +53,33 @@ <h3 *ngIf="addressConfiguration.type === 'editWarehouse'">{{ localeData?.edit_wa
                                         </div>
 
                                         <div class="row">
-                                            <div *ngIf="addressConfiguration?.tax?.name" class="col-sm-4 form-group">
+                                            <div *ngIf="isBusinessTypeRegistered" class="col-sm-4 form-group">
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/shared/create-address/create-address.component.html:56]: confirm with @ravindersinghdang sir

File: apps/web-giddh/src/app/theme/form-fields/reactive-dropdown-field/reactive-dropdown-field.component.ts [hunk 6]
```diff
@@ -172,6 +172,10 @@ export class ReactiveDropdownFieldComponent implements ControlValueAccessor, OnI
         if (changes.openDropdown?.currentValue && !changes?.openDropdown?.previousValue) {
             this.openDropdownPanel();
         }
+
+        if (changes.labelValue) {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/theme/form-fields/reactive-dropdown-field/reactive-dropdown-field.component.ts]:
```suggestion
        if (changes?.labelValue) {
```

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.ts]: Done

File: apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts:76]: Checked

File: apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts (hunk 3 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts]: Done

File: apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts (hunk 4 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts:94]: Write this in ngAfterViewInit and removed ngOnchanges

File: apps/web-giddh/src/app/theme/form-fields/reactive-dropdown-field/reactive-dropdown-field.component.ts (hunk 6 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/theme/form-fields/reactive-dropdown-field/reactive-dropdown-field.component.ts]: Done

File: apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts [hunk 7]
```diff
@@ -80,12 +80,12 @@ export class PersonalInformationComponent implements OnInit, OnChanges, OnDestro
...
     *
     * @memberof PersonalInformationComponent
     */
-    public ngOnChanges(changes: SimpleChanges): void {
-        if (changes?.profileData) {
+    public ngAfterViewInit(): void {
+        if (this.profileData) {
             if (this.profileData?.alias || this.profileData?.name) {
                 this.initProfileForm(this.profileData);
             }
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts:91]: confirm with @ravindersinghdang sir

File: apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts (hunk 2 above)
Comment (dilpreetsinghofficial) [reply apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts:76]: confirm with @ravindersinghdang sir for this condition

File: apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts (hunk 4 above)
Comment (dilpreetsinghofficial) [reply apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts:94]: confirm with @ravindersinghdang sir for this condition

**Review by dilpreetsinghofficial — APPROVED** [review]

File: apps/web-giddh/src/app/shared/create-address/create-address.component.html (hunk 5 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/shared/create-address/create-address.component.html:56]: @dvCodeWorld this won't work if the index of tax number is 0.

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**